pilot/pkg/bootstrap/server.go

```diff
@@ -1128,7 +1136,12 @@ func (s *Server) grpcServerOptions() []grpc.ServerOption {
 		prometheus.UnaryServerInterceptor,
 	}
 
+	// EnableHandlingTimeHistogram() has a data race within it (reading/writing to its
+	// serverHandledHistogramEnabled). Using a package mutex to avoid data races.
+	// Should be re-examined with an updated revision of the go-grpc-prometheus.
+	globalMutex.Lock()
```

Comment:
How about removing this, and use `sync.Once`

Reply:
Good idea. Will replace with sync.Once.